api/package.json

```diff
@@ -17,6 +17,15 @@
   "bugs": {
     "url": "https://github.com/DashboardHub/PipelineDashboard/issues"
   },
+  "scripts": {
+    "install:local": "npx serverless dynamodb install --stage dev",
```

Comment:
don't need any `npx`in package.json - all commands here assume usage of the installed modules automatically